detect_secrets/plugins/base.py

```diff
@@ -71,10 +70,7 @@ def json(self) -> Dict[str, Any]:
             'name': self.__class__.__name__,
         }
 
-    def format_scan_result(self, secret: Optional[PotentialSecret]) -> str:
-        if not secret:
-            return 'False'
-
+    def format_scan_result(self, secret: PotentialSecret) -> str:
```

Comment:
can you explain what changed to make `None` no longer an allowed arg here? having trouble seeing it

Reply:
This was my realization that this is never called with `secret=None`. This function is called [here](https://github.com/Yelp/detect-secrets/blob/pre-v1-launch/detect_secrets/main.py#L94), which only passes values in that are actual secrets.

Additionally, we never really have an explicit `None` value to pass into this function. `analyze_line` gives us a set of `PotentialSecret`s, so it doesn't cohere with the other functionality to obtain a `None` value to format.